components/typography/metadata/typography-international-deprecated.yml

```diff
@@ -1,12 +1,12 @@
-name: Typography (Int., Deprecated I)
+name: Typography (Internationalized, Deprecated)
 status: Deprecated
+hide: true
```

Comment:
are we sure we want to hide the old typography?

Reply:
It's very, very confusing otherwise. We link to the deprecated typography from the landing page, so it's not orphaned.